src/main/java/dev/agones/AgonesSDK.java

```diff
@@ -64,7 +64,7 @@ public void startHealthTask(int period, TimeUnit unit) {
     }
 
     public void health() {
-        getEndpoints(AgonesEndpoints.class).health();
+        getEndpoints(AgonesEndpoints.class).health(new Object());
```

Comment:
An `Empty` model class would be better here. (See models package)

Reply:
Which `Empty` model are you talking about?